src/application/service_tester.go

```diff
@@ -88,11 +88,13 @@ func (s *ServiceTester) getDockerComposeConfig() (types.DockerCompose, error) {
 	if err != nil {
 		return result, err
 	}
-	dockerConfigs := types.DockerConfigs{}
-	serviceDockerConfig := appDockerConfigs[s.Role]
-	serviceDockerConfig.DependsOn = s.getDependencyContainerNames()
-	serviceDockerConfig.Command = s.ServiceConfig.Development.Scripts["test"]
-	dockerConfigs[s.Role] = serviceDockerConfig
+	dockerConfigs, err := s.Initializer.getServiceDockerConfig(s.Role, s.ServiceConfig, types.ServiceData{Location: s.Role})
```

Comment:
Why would `Location` be `s.Role` here?

Reply:
The role in this case is the service dir (we need your rename pr in)